src/modules/windowwalker/dll/WindowWalker.rc

```diff
@@ -19,12 +19,12 @@ BEGIN
         BLOCK "040904b0"
         BEGIN
             VALUE "CompanyName", COMPANY_NAME
-            VALUE "FileDescription", "$projectname$ Module"
+            VALUE "FileDescription", "WindowWalker PowerToy Module"

```

Comment:
Branding review needed here. @crutkas are these *PowerToys* Modules or *PowerToy* Modules? I'd say the former but will defer.

Reply:
Let's open an issue for this since I would prefer to fix it for all modules and not just for this one and it's not a high priority to stop this PR to be merged.